bitgouel-api/src/main/kotlin/team/msg/domain/university/exception/AlreadyExistUniversityNameException.kt

```diff
@@ -0,0 +1,8 @@
+package team.msg.domain.university.exception
+
+import team.msg.domain.university.exception.constant.UniversityErrorCode
+import team.msg.global.error.exception.BitgouelException
+
+class AlreadyExistUniversityNameException(
```

Comment:
```suggestion
class AlreadyExistUniversityException(
```
이름이 같은 대학교가 존재할 수 없으니 위처럼 작성하는 게 더 좋아보이는데, 어떤가요?

Reply:
https://github.com/School-of-Company/Bitgouel-Server/pull/491/commits/be187a2e3899d6e4ce756ccb7f6d6a944d4a1494

적용했습니다 suggestion은 저 이름이 바뀌면 여러 곳에서 수정된 이름이 자동 반영이 안되서 하지 않은거에요 서운해하지 말아주세용

Reply:
네네 인지하고 있습니다! 👍